source/documentation/rules-g.html.md.erb

```diff
@@ -0,0 +1,1208 @@
+
+<b>Technical Description</b>
+
+N/A
+
+##G0014
+
+<b>Functional Description</b>
+
+Eastern longitude and Northern latitude will use the optional '+' sign.<br/>Western longitude and Southern latitude will use the '-' sign.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0015
+
+<b>Functional Description</b>
+
+In case of Incident information received via one or multiple CD180C, all the information received shall be included in the D.G. &lt;CONSIGNMENT-INCIDENT&gt;.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0016
+
+<b>Functional Description</b>
+
+&lt;CONSIGNMENT-INCIDENT-TRANSPORT EQUIPMENT.Container identification number&gt; is filled in case of incident with the new Container identification number amending initial declaration or with the existing Container identification number if DI Number of seals is greater than 0.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0017
+
+<b>Functional Description</b>
+
+'State of seals' = 1 in case that Seals are in good state (i.e. present and not damaged)<br/>'State of seals' = 0 in case that Seals are not in good state (i.e. not present or damaged).
+
+<b>Technical Description</b>
+
+N/A
+
+##G0018
+
+<b>Functional Description</b>
+
+IF &lt;COMPETENT CUSTOMS OFFICE AT DEPARTURE&gt; is EQUAL to &lt;CUSTOMS OFFICE OF DEPARTURE&gt;<br/>THEN &lt;CD144C-CUSTOMS OFFICE OF DEPARTURE&gt; = "O" <br/>ELSE &lt;CD144C-CUSTOMS OFFICE OF DEPARTURE&gt; = "R"
+
+<b>Technical Description</b>
+
+N/A
+
+##G0020
+
+<b>Functional Description</b>
+
+Refers to the mode of transport corresponding to the active means of transport which is expected to be used on exit from or entry into the Safety and Security area.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0021
+
+<b>Functional Description</b>
+
+The value '0' (zero) is a valid number in this Data Item, as per applicable XSD pattern.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0022
+
+<b>Functional Description</b>
+
+This D.I. corresponds to the D.I. Security of the Export Declaration.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0023
+
+<b>Functional Description</b>
+
+The D.I. will be filled in with new value amending initial declaration in case of incident
+
+<b>Technical Description</b>
+
+N/A
+
+##G0024
+
+<b>Functional Description</b>
+
+D.I. will be filled if the value is provided.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0025
+
+<b>Functional Description</b>
+
+The value of D.I. &lt;CONSIGNMENT-HOUSE CONSIGNMENT.Security indicator from export declaration&gt; will be equal to D.I. &lt;CC191C-AES RESULTS-EXPORT OPERATION.Security&gt;.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0026
+
+<b>Functional Description</b>
+
+This D.G./D.I. can be used only  <br/>         - in case of Export followed by transit (maximum one MRN included per one House Consignment) <br/>     AND <br/>         - if the &lt;TRANSIT OPERATION.Additional Declaration Type&gt; is EQUAL to 'A'.
+
```

Comment:
Again, moving suddenly into an abbreviation - `D.G.`

Will this be clear to novice readers?